### DISCUSSION TITLE
apps내의 volunteer, shelter내의 page가 아닌 components 구조에 대해

### DISCUSSION THREAD

**Discussion by sukvvon** [Anifriends/Anifriends-Frontend#72] [General]
`shared` 내부의 `components`에 공통 컴포넌트들을 보관하고 있습니다.
아직 `apps` 내부에 `components` 폴더가 존재하지 않아서 어떻게 관리하면 좋을지에 대해 의논하고자 합니다.

봉사자(volunteer), 보호소(shelter) 내부에서도 `shared`와 같이 `components` 폴더를 만들어서 내부에 `VolunteerProfile.tsx`처럼 tsx 파일들을 놓는 것이 어떨까요? 

이미 `shared`에 공통 `components`들을 다 정리해놓아서 `apps` 내부에서는 파일의 개수가 그렇게 많지 않을 것 같다는 생각입니다.

**Comment by DongjaJ** [comment]
components 폴더를 만들긴 해야 할것 같은데 
1. app내에서 공통으로 사용할 컴포넌트들을 넣고 
2. 페이지에서만 사용하는 컴포넌트들은 페이지 디렉토리에 컴포넌트를 만드는게 어떨까합니다

**Reply by kutta97** [reply]
괜찮은 생각인 것 같습니다

**Reply by sukvvon** [reply]
> 1. app내에서 공통으로 사용할 컴포넌트들을 넣고

apps/shelter/src/components 폴더를 만들고 공통적으로 사용이 될 것 같은 경우 components 폴더 내에 컴포넌트를 생성하자는 말씀이신가요? 

> 2. 페이지에서만 사용하는 컴포넌트들은 페이지 디렉토리에 컴포넌트를 만드는게 어떨까합니다

페이지에서만 사용할 줄 알고 페이지 디렉토리에 컴포넌트를 만들었다가 다른 페이지에서 사용할 수 있다는 것을 알게되면 어떻게 하나요? apps/shelter/src/components 폴더 내부로 이동시킨 후 이동이 된 컴포넌트들 path를 모두 수정하나요?
